src/components/d-active-card/dActiveCard-styles.ts

```diff
@@ -0,0 +1,49 @@
+// libraries
+import { StyleSheet } from "react-native";
+
+// theme
+import { Typography } from "../../theme/typography";
+import { COLORS } from "../../theme/color";
+import { SPACING } from "../../theme/spacing";
+
+export const styles = StyleSheet.create({
+    container:{
+        flex:SPACING.space_1,
+        flexDirection:'column',
+        justifyContent:"center",
```

Comment:
use prettier

Reply:
I wrote the code as provided by you in the doc for prettier.js in the file but its not working.